Makefile

```diff
@@ -62,7 +63,9 @@ docker-test-unit: $(DOCKER_CACHE)
 
 .PHONY: docker-tidy
 docker-tidy: $(DOCKER_CACHE)
+	@rm -f go.sum
 	@$(DOCKER_ENV) go mod tidy
+	@chown $$(id -u):$$(id -g) go.mod go.sum
```

Comment:
wouldn't this fail if `make docker-tidy` is not run as root?

Reply:
My fault, I have changed to add `-u $UID:$GID` to `DOCKER_ENV`.